test/test_psl.c

```diff
@@ -229,11 +298,11 @@ START_TEST(test_dump)
    tb_rewind(tb);
 
    psl_dump(tree_psl(tree_stmt(a, 1)));
-   ck_assert_str_eq(tb_get(tb), "assert never B");
+   ck_assert_str_eq(tb_get(tb), "assert never {B}");
```

Comment:
This doesn't look right: a Boolean in an FL property shouldn't be parsed as a SERE.